app/serializers/person_escort_record_serializer.rb

```diff
@@ -19,6 +18,14 @@ def framework_responses
     object.framework_responses.includes(:flags, framework_question: :framework)
   end
 
+  def status
+    if object.state == 'unstarted'
+      'not_started'
```

Comment:
we cannot use `not_started` in the enum as negative scopes and enum elements prefixed with "not_" will cause a warning, so only surface here

Reply:
Presumably the state is never sent back to the back end in a request, so one way conversion suffices?

Reply:
Discussed offline, front end never sends this attribute back.